$OpenBSD: patch-misc,v 1.1.1.1 2000/09/05 11:37:29 espie Exp $

- change arg type for gmtime() and localtime() from long to time_t


--- misc.h.orig	Sat Dec 18 17:06:39 1999
+++ misc.h	Wed Apr  8 00:34:38 2009
@@ -90,7 +90,7 @@ char *CanonicaliseName(char *name);
 
 char *MakeHash(const char *args);
 
-char *RFC822Date(long t,int utc);
+char *RFC822Date(time_t t,int utc);
 long DateToTimeT(const char *date);
 
 char *URLDecode(const char *str, int isform);
